tests/tests/test_fault_tolerance.py

```diff
@@ -32,10 +32,6 @@
 from testutils.infra.container_manager.kubernetes_manager import isK8S
 
 DOWNLOAD_RETRY_TIMEOUT_TEST_SETS = [
-    {
-        "blockAfterStart": False,
-        "logMessageToLookFor": "[uU]pdate (check|fetch) failed:",
-    },
```

Comment:
Removed unintentionally?

Reply:
No :)

Reply:
I think I know why, it's because the old fetch and store steps in the Golang client are done in the same state now, right? I think this deserves a "why" in the commit message though. I also found this confusing.

Reply:
Should fix this, but otherwise looks good to me.

Reply:
Yes, should have been a separate commit, splitting it out.